src/global.d.ts

```diff
@@ -0,0 +1 @@
+interface EmscriptenModule {}
```

Comment:
Needed for the type checker to be happy with the optional init argument we don't use. I didn't want to install this entire dependency if we're not going to use it.